src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/services/PrisonersInPrisonService.kt

```diff
@@ -139,7 +138,7 @@ class PrisonersInPrisonService(
       .type(MultiMatchQueryBuilder.Type.CROSS_FIELDS)
       .operator(Operator.AND)
 
-    val termsList = term.split("\\s".toRegex()).map { it.uppercase() }
+    val termsList = term.split("[\\s,;:+]".toRegex()).map { it.uppercase() }
```

Comment:
I could just added a comma, but added a few more for good measure but not punctuation that can appear in names; hyphen etc